### ISSUE TITLE
Training issues

### ISSUE THREAD

**Issue by Im-fengyin** [SimingYan/HPNet#14]
Thanks for the code, and the great work. Hope you can explain some training details.
1. Whether data augmentation was used during training, the training setting in readme.md is different with supplementary Material.
2. Are there any setting that have a huge impact on the results? I rebuild the network and trained it for 100 epoches with initial lr 1e-3 and decay step 40, but both $L_{emb}$ and $L_{nnl}$ are worse than the provided pretrained model on the test set.
Thanks for answer

**Comment by SimingYan** [comment]
1. We have data augmentation during the training. But this should not be the big impact.
2. If you are training on the same dataset, the loss should be very stable. Could you please provide more details of training? I think your training schedule is the same as ours. And how about the mIoU result?

**Comment by Im-fengyin** [comment]
> 

Thanks for the answering. I found that my code made a mistake in KNN. Now I train the network again.

**Comment by SimingYan** [comment]
Good to hear that! Please let me know if there is still any problem.
